interface.js

```diff
@@ -0,0 +1,436 @@
+console.log('interface.js loaded');
+const Store = require('electron-store');
```

Comment:
it looks like nodejs imports are fine here?

Reply:
Indeed! Not sure what was going on the other day — imports are working fine so i split out each provider class to it's own file.